object_detection/detectron2_training.ipynb

```diff
@@ -0,0 +1,243 @@
+{
+ "cells": [
```

Comment:
need requirements.txt file

Reply:
need a README inside the object_detection folder that in 1-line describes what is difference between the two notebooks